server/data-import/data.js

```diff
@@ -25,23 +22,36 @@ loadData()
 
 // note that exports are 'mongoexport --db hah-dev --collection pregnancycenters --jsonArray --out cessilye_nypc_geocoded.json'
 
-PregnancyCenterModel.collection.dropAsync()
-	.catch( (err) => log.error(err))
-	.then( () => {
-		fs.readFile('test/fixtures/cessilye_nypc.json', 'utf8', (err, data) => {
-			const docs = EJSON.parse(data)
-			log.info(typeof docs)
-			for (const doc of docs) {
+async function reimport() {
 
-				Joi.validateAsync(doc, pregnancyCenterSchemaJoi, {abortEarly: false})
-					.catch( (err) => log.error(err))
-					.then( (validatedData) => {
-						PregnancyCenterModel.createAsync(validatedData)
-							.then( (pc) => log.info(pc))
-					})
+	let docs
 
-			}
+	try {
+		await PregnancyCenterModel.collection.drop()
```

Comment:
If this was user facing, I would say try/catch with an exit code and a specific error message that is user friendly. But because it's a script that only devs will run, I think taking out the try/catch would be best. It looks cleaner IMO. In general, try/catches are useful when you want to change something about the error that is shown or add info to it to help in debugging. In this case though, we just want it to fail and print out the error message in the console (which happens automatically).